drag_and_drop_v2/translations/eo/LC_MESSAGES/text.po

```diff
@@ -521,6 +521,14 @@ msgstr "Pläçéd ïn: {zone_title}  Ⱡ'σяєм ιρѕυм ∂σłσя ѕιт
 msgid "Correctly placed in: {zone_title}"
 msgstr "Çörréçtlý pläçéd ïn: {zone_title} Ⱡ'σяєм ιρѕυм ∂σłσя ѕιт αмєт, #"
 
+#: drag_and_drop_v2/public/js/drag_and_drop.js
+msgid ", draggable, grabbed"
```

Comment:
@cahrens Yes - at the end of the README there are [two sections about translations](https://github.com/edx-solutions/xblock-drag-and-drop-v2#i18n-compatibility).

Spoiler: it is far from ideal, but we were unable to come up with anything that does not require significant updates to i18n_tool.